src/Shell/AclShell.php

```diff
@@ -367,7 +367,10 @@ public function view()
      */
     public function initdb()
     {
-        return $this->dispatchShell('schema create DbAcl');
+        $_SERVER['argv'][1]='migrations';
+        $_SERVER['argv'][2]='migrate';
+        $_SERVER['argv'][3]='--plugin=acl';
```

Comment:
I wonder if this is a quirk of migrations shell being implemented mostly as extensions to Phinx.